### PR TITLE
Add func sai_thrift_clear_buffer_pool_watermark

### PR DESCRIPTION
**Pull request by zhixzhu**
### Description of PR

For buffer pool watermark case, need a func to clear watermark, then next read will get more accurate watermark.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### CODE REVIEW

File: tests/saitests/py3/switch.py [hunk 1]
```diff
@@ -906,6 +906,15 @@ def sai_thrift_read_buffer_pool_watermark(client, buffer_pool_id):
     return wm_vals[0]
 
 
+def sai_thrift_clear_buffer_pool_watermark(client, buffer_pool_id):
```
Comment (neethajohn) [inline tests/saitests/py3/switch.py:909]: where is this func used?

Comment (zhixzhu) [reply tests/saitests/py3/switch.py:909]: Planned to use in class BufferPoolWatermarkTest.

The new RPC API is merged in https://github.com/opencomputeproject/SAI/pull/1804, I still need to figure out how SONiC get this update before moving forward.

**Comment by wangxin** [comment]
@zhixzhu Is this a backport of master branch PR to 202205 branch? Can you mention the master branch PR ID here in PR description?

**Comment by wangxin** [comment]
@zhixzhu Then can you close this PR and create a PR to the master branch instead. After the PR to master branch is merged, then we can cherry-pick it to 202205 branch.

**Pull request closed without merging**